package.json

```diff
@@ -59,7 +59,7 @@
     "@jupyterlite/server": "^0.1.0-alpha.14"
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.1.0",
+    "@jupyterlab/builder": "^3.4.1",
```

Comment:
This looks fine. Although normally the ranges should be compatible, so maybe there is an issue with the check in JupyterLab. iirc the cookiecutter still uses `^3.1.0` and builds fine.

Or the `yarn.lock` includes multiple `@jupyterlab/builder`. It would be fine to delete it and recreate it at this stage (or deduplicate).

Reply:
The weird thing is that yesterday it was building just fine, the error appeared this morning.

If we trigger the CI on the cookiecutter we might be able to replicate the error?

Reply:
Let's see https://github.com/jupyterlab/extension-cookiecutter-ts/pull/218

Reply:
Eric released 3.4.1 this morning could it be that the CI just ran during the publication with only a subset of npm packages published hence some errors?

Reply:
Makes sense 👍🏽

Reply:
Restarting the job on `main` to double check: https://github.com/jupyterlite/xeus-python-kernel/actions/runs/2311848838

Reply:
Still fails with 
```
ValueError: Extensions require a devDependency on @jupyterlab/builder@^3.4.1, you have a dependency on 3.4.0
```